solidity/contracts/hooks/igp/InterchainGasPaymaster.sol

```diff
@@ -218,6 +254,22 @@ contract InterchainGasPaymaster is
         return _gasOracle.getExchangeRateAndGasPrice(_destinationDomain);
     }
 
+    /**
+     * @notice Returns the stored destinationGasOverhead added to the _gasAmount.
+     * @dev If there is no stored destinationGasOverhead, 0 is used.
```

Comment:
should we have an immutable default? or revert?

Reply:
added more context